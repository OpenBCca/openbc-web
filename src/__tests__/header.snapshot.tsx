import Header from '@/components/header/header';
import { render } from '@testing-library/react';

// Use this if you want to skip this test
describe.skip('skips snapshot test', () => {});

describe('Header snapshot test', () => {
  it('should render header unchanged', () => {
    const { container } = render(<Header />);
    expect(container).toMatchSnapshot();
  });
});
