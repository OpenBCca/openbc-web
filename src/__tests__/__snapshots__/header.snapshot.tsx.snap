// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header snapshot test should render header unchanged 1`] = `
<div>
  <header
    class="MuiBox-root css-sdwowb"
  >
    <div
      class="MuiBox-root css-1s56th"
    >
      <div
        class="MuiBox-root css-ziu5zu"
      >
        OpenBC
      </div>
      <div
        class="MuiBottomNavigation-root css-w3uaz-MuiBottomNavigation-root"
      >
        <button
          class="MuiButtonBase-root MuiBottomNavigationAction-root Mui-selected css-13m47w6-MuiButtonBase-root-MuiBottomNavigationAction-root"
          tabindex="0"
          type="button"
        >
          <span
            class="MuiBottomNavigationAction-label Mui-selected css-imwso6-MuiBottomNavigationAction-label"
          >
            Projects
          </span>
          <span
            class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
          />
        </button>
        <button
          class="MuiButtonBase-root MuiBottomNavigationAction-root css-13m47w6-MuiButtonBase-root-MuiBottomNavigationAction-root"
          tabindex="0"
          type="button"
        >
          <span
            class="MuiBottomNavigationAction-label css-imwso6-MuiBottomNavigationAction-label"
          >
            Join Us
          </span>
          <span
            class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
          />
        </button>
        <button
          class="MuiButtonBase-root MuiBottomNavigationAction-root css-13m47w6-MuiButtonBase-root-MuiBottomNavigationAction-root"
          tabindex="0"
          type="button"
        >
          <span
            class="MuiBottomNavigationAction-label css-imwso6-MuiBottomNavigationAction-label"
          >
            About
          </span>
          <span
            class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
          />
        </button>
      </div>
    </div>
  </header>
</div>
`;
